pkgs/by-name/sd/SDL2_classic_mixer_2_0/package.nix

```diff
@@ -0,0 +1,8 @@
+{
+  SDL2_mixer_2_0,
+  SDL2_classic_mixer,
+}:
+
+SDL2_mixer_2_0.override {
+  SDL2_mixer = SDL2_classic_mixer;
+}
```

Comment:
is `SDL2_mixer_2_0` actually needed? The analogous `SDL_image_2_0` was unnecessary, see https://github.com/NixOS/nixpkgs/pull/400338/commits/8375464933954bcf54e05f979a3d2aeca0bd2632

Reply:
sounds like a separate pr

Reply:
fair i guess